### PR TITLE
Tweaks the new christmas event

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
The tree will simply spawn with presents under it since everyone hanging around the bar for 10 minutes waiting on the spawn is silly

The christmas event message will now inform people of said presents

Changed up the paths of christmas trees since inheriting the random icon stuff from pine trees made it messy and they don't actually have any in common with them save for a dmi

### CODE REVIEW

**Comment by coiax** [comment]
Are you going to keep trees keeping desperate ckey lists?

**Comment by Tlaltecuhtli** [comment]
>people had a reason to hangout at the bar for 10 min

File: code/modules/events/holiday/xmas.dm [hunk 1]
```diff
@@ -65,17 +60,23 @@
...
 	name = "christmas tree spawner"
-	var/tree = /obj/structure/flora/tree/pine/xmas
 
 /obj/effect/landmark/xmastree/Initialize(mapload)
-	..()
-	if(FESTIVE_SEASON in SSevents.holidays)
-		new tree(get_turf(src))
+	. = ..()
+	if(CHRISTMAS in SSevents.holidays)
+		new /obj/structure/flora/tree/xmas/presents(get_turf(src))
```
Comment (Cyberboss) [inline code/modules/events/holiday/xmas.dm]: loc

File: code/modules/events/holiday/xmas.dm [hunk 2]
```diff
@@ -65,17 +60,23 @@
...
+	else if(FESTIVE_SEASON in SSevents.holidays)
+		new /obj/structure/flora/tree/xmas(get_turf(src))
```
Comment (Cyberboss) [inline code/modules/events/holiday/xmas.dm]: loc

File: code/modules/events/holiday/xmas.dm [hunk 3]
```diff
@@ -65,17 +60,23 @@
...
 
-/obj/effect/landmark/xmastree/rdrod
+/obj/effect/landmark/rdrod
 	name = "festivus pole spawner"
-	tree = /obj/structure/festivus
+
+/obj/effect/landmark/rdrod/Initialize(mapload)
+	. = ..()
+	if(FESTIVE_SEASON in SSevents.holidays)
+		new /obj/structure/festivus(get_turf(src))
```
Comment (Cyberboss) [inline code/modules/events/holiday/xmas.dm]: loc

**Review by Cyberboss — CHANGES_REQUESTED** [review]
You should remove the event or force it to run since it's pointless now

**Comment by ShizCalev** [comment]
Compile.

**Comment by KorPhaeron** [comment]
Not gonna finish this before the event is over

**Comment by coiax** [comment]
That doesn't stop you from preparing for NEXT YEARS CHRISTMAS! WOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Pull request closed without merging**